containers/support/BugModal.js

```diff
@@ -0,0 +1,183 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { reportBug } from 'proton-shared/lib/api/reports';
+import { getOS, getBrowser, getDevice } from 'proton-shared/lib/helpers/browser';
+import { c } from 'ttag';
+import {
+    Modal,
+    Href,
+    Alert,
+    Row,
+    Input,
+    Button,
+    useToggle,
+    Info,
+    TextArea,
+    Label,
+    ContentModal,
+    FooterModal,
+    EmailInput,
+    ResetButton,
+    PrimaryButton,
+    useApiWithoutResult,
+    useUser,
+    useAddresses,
+    useNotifications,
+    useConfig
+} from 'react-components';
+import AttachScreenshot from './AttachScreenshot';
+
+const CONFIGS = {
+    Web: 'Angular',
+    ProtonMailReact: 'ProtonMail React',
+    ProtonContactReact: 'ProtonContact React',
+    ProtonMailSettingsReact: 'ProtonMail Settings React',
+    ProtonCalendarReact: 'ProtonCalendar React',
+    ProtonDriveReact: 'ProtonDrive React',
+    ProtonWalletReact: 'ProtonWallet React',
+    ProtonVPN: 'ProtonVPN React'
+};
+
+const BugModal = ({ show, onClose }) => {
+    const { CLIENT_ID, APP_VERSION } = useConfig();
+    const Client = CONFIGS[CLIENT_ID];
+    const [{ Name = '' }] = useUser();
+    const { createNotification } = useNotifications();
+    const [addresses = []] = useAddresses();
+    const [{ Email = '' } = {}] = addresses;
+    const os = getOS();
+    const browser = getBrowser();
+    const device = getDevice();
+    const [model, update] = useState({
```

Comment:
imho the defaut object should come from a helper, not as a default inside the modal